lighting/common/perceptual2LinearRoughness.glsl

```diff
@@ -0,0 +1,20 @@
+/*
+contributors: Shadi El Hajj
+description: Classic Disney reparametrisation of roughness from Burley's paper "Physically Based Shading At Disney". Sébastien Lagarde's recommends clamping perceptual roughness to 0.045 in his paper "Moving Frostbite to PBR".
+use: <float> perceptual2LinearRoughness(<float> perceptualRoughness)
+license: MIT License (MIT) Copyright (c) 2024 Shadi EL Hajj
+*/
+
+#ifndef MIN_PERCEPTUAL_ROUGHNESS
+#define MIN_PERCEPTUAL_ROUGHNESS 0.045
+#endif
+
+#ifndef FNC_PERCEPTUAL_LINEAR_ROUGHNESS
+#define FNC_PERCEPTUAL_LINEAR_ROUGHNESS
+
+float perceptual2LinearRoughness(float perceptualRoughness) {
```

Comment:
NIT: suggesting `perceptual2linearRoughtness()` other cases where we use `2` the inmediate letter is lowercase.

Reply:
Fixed :-)